tests/UI/campaigns/functional/BO/03_catalog/01_products/01_filterAndQuickEditProducts.js

```diff
@@ -132,7 +132,7 @@ describe('Filter Products', async () => {
     ];
 
     tests.forEach((test) => {
-      filterValue = test.args.filterValue.min === undefined ? `'${test.args.filterValue}`
+      filterValue = test.args.filterValue.min === undefined ? `'${test.args.filterValue}'`
```

Comment:
That didn't break before ?

Reply:
just something to display as Scenario name, will not break anything